src/wp-content/themes/twentytwentytwo/templates/page.html

```diff
@@ -17,9 +17,9 @@
 
 <!-- wp:post-content {"layout":{"inherit":true}} /-->
 
-<!-- wp:group {"layout":{"inherit":true}} -->
+<!-- wp:group {"layout":{"inherit":true},"style":{"spacing":{"padding":{"top":"var(--wp--custom--spacing--medium, 6rem)"}}}} -->
 <div class="wp-block-group">
```

Comment:
The spacing needs to be added to the element, not only in the comment.

Reply:
good catch!!!